kitsune/questions/views.py

```diff
@@ -519,14 +519,18 @@ def aaq(request, product_key=None, category_key=None, showform=False,
     if product_key is None:
         product_key = request.GET.get('product')
         if request.MOBILE and product_key is None:
+            ua = request.META.get('HTTP_USER_AGENT', '').lower()
+
             # Firefox OS is weird. The best way we can detect it is to
             # look for a mobile Firefox that is not Android.
-            ua = request.META.get('HTTP_USER_AGENT', '').lower()
             if 'firefox' in ua and 'android' not in ua:
                 product_key = 'firefox-os'
+            elif 'rocket' in ua:
```

Comment:
will the above `if` statement not catch rocket first, or does it report "android"?

Reply:
also are they going to keep "rocket" in the UA for the foreseeable future, or will they move to "lite"?

Reply:
Just posted the UA string example I received in the opening comment. It shows `firefox` is *not* present, but `android` is, so I believe this logic will work as expected.